herbert_core/classes/instrument_classes/instrument/detectors/@IX_detector_array/IX_detector_array.m

```diff
@@ -331,11 +446,12 @@
 
         end
     end
-    
+    %}
```

Comment:
To be cleared in some future version (also others)?

Reply:
Yes to be cleared after the merge